src/Pairse_elements.py

```diff
@@ -0,0 +1,36 @@
+# Пары элементов
+# Дан список чисел. Посчитайте, сколько в нем пар элементов,
+# равных друг другу. Считается, что любые два элемента, равные
+# друг другу образуют одну пару, которую необходимо посчитать.
+# Входные данные - строка из чисел, разделенная пробелами.
+# Выходные данные - количество пар.
+# Важно: 1 1 1 - это 3 пары, 1 1 1 1 - это 6 пар
+
+my_list = [int(i) for i in input('Введите список: ').split()]
+count_pairs = 0
+list_replay_num = []
+for i in my_list:
+    if my_list.count(i) > 0 and i not in list_replay_num:
+        list_replay_num.append(i)
+        for j in range(my_list.count(i)):
+            count_pairs += j
+print(count_pairs)
+
+# Решение №2. Использовал кусок кода из занятия с dict.
+# С каждым увелечением пар, разница между числами sum_pairs
+# увеличивается на 1.
+
+first_list = [int(i) for i in input('Введите список: ').split()]
+dct = {}
+for i in first_list:
```

Comment:
it consists only of the same numbers so here list.count(list[0]) would have the same complexity